jobs/delorean/jenkinsfiles/branch/ga/Jenkinsfile

```diff
@@ -108,6 +157,12 @@ node {
                 String body = "Created automatically by jenkins"
                 GHPullRequest pr = ghFindOrCreatePullRequest(gitHub.getRepository("${ghOwner}/${ghRepo}"), "${ghOwner}:${sourceBranch}", targetBranch, title, body)
                 println "Component Update PR = ${pr.getHtmlUrl()}"
+                def (found, key) = hasJiraUpdateIssue(credentials, productName)
+                if (!found) {
+                    createJiraIssue(jiraUserPass, pr.getHtmlUrl(), productName)
+                } else {
+                    println "[INFO] Issue already open for ${productName}: https://issues.jboss.org/browse/${key}"
+                }
```

Comment:
Should we move this into it's own "Create Jira" stage, or is there a reason it needs to be in here?

Reply:
Ah we use the pr object

Reply:
we only do it if a pr is going to be sent